src/EditorFeatures/Core/LanguageServer/Handlers/Symbols/DocumentSymbolsHandler.cs

```diff
@@ -53,7 +54,8 @@ public async Task<object[]> HandleRequestAsync(DocumentSymbolParams request, Req
             // TODO - Return more than 2 levels of symbols.
```

Comment:
> What do thsee numbers represent:
> 
> ![image](https://user-images.githubusercontent.com/4564579/172740014-d0aac563-70fe-42c3-bbfd-1b5ded25c28c.png)
> 
> ?

Definitely should have clarified earlier, @CyrusNajmabadi it represents the number of children a node should have (but I do not plan on keeping it forever). I would like to keep it until I can test getting more than 2 levels of symbols (I've been told this change is a WIP?)